app/Http/Controllers/HomeController.php

```diff
@@ -61,7 +66,56 @@ public function index()
                 'articlesRemainingCount',
                 'collections',
                 'collectionsRemainingCount',
+                'countsBlurb',
             ])
         );
     }
+
+    private function getCountsBlurbData()
+    {
+        [$itemsCount, $choices] = Cache::remember('home.counts', now()->addDays(3), function () {
+            $galleriesCount = 15;
+            $authoritiesCount = Authority::count();
+            $itemsCount = Item::count();
+            $highResItemsCount = Item::withExists('images')->count();
+
+            $itemRepository = app(ItemRepository::class);
+            $freeItemsCount = $itemRepository->count((new ItemFilter())->setIsFree(true));
+
+            return [
+                $itemsCount,
+                [
+                    [
+                        trans('intro.from_galleries_start'),
+                        route('frontend.info'),
+                        $galleriesCount,
+                        trans('intro.from_galleries_end'),
```

Comment:
I understand it comes up from the legacy code. But now I feel I am missing keys in this array. 
Also I would prefer to see `trans` used in view, not in controller. 
Something like

```suggestion
                        'claim_start' => 'intro.from_galleries_start',
                        'url' => route('frontend.info'),
                        'count' => $galleriesCount,
                        'claim_end' => 'intro.from_galleries_end',
```

Reply:
Yeah, I agree. I just wanted to re-use the old solution but it was kinda confusing. Changed now.